README.md

```diff
@@ -15,14 +17,20 @@ Detailed documentation can be found in our [GitHub Documenation Page](http://vmw
 
 ## Download
 
-**[Click here to download (Github releases)] (https://github.com/vmware/docker-volume-vsphere/releases)**
+**Click here to download [Github releases](https://github.com/vmware/docker-volume-vsphere/releases)**
```

Comment:
Github release page still shows the change log and release note. Deliverable will be removed and bintray link for VIB download will be provided from Github release page.

what do you suggest?